### PR TITLE
Readme/diagram

### PR DESCRIPTION
**Pull request by Nazar865**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+https://drive.google.com/file/d/1OogCU_qmvH-yhxRAk1pWMAEwSOxFxhkL/view?usp=sharing
```
Comment (ghost) [inline README.md]: - **User table**: User can have many cards, but in the same time a card can have only one host, relation is good, but User table configuration has wrong prop: FK CardId. If you have one-to-many relation, FK is needed only on many side (as it is: UserId), just remove the CardId from user table.
-  **Transfer table**: To my mind, to & from CardId looks not bad, but senderCardId and receiverCardId or something like this is better.
- **Transfer - Card relation**: If u have many-to-many relation, u should create the third table to handle, check [here](https://www.datensen.com/blog/er-diagram/many-to-many-relationships/).
- **Deposit - Card relation**: same as above
- **Currency - Card relation**: many-to-many? why? your card have only one currency, should be one-to-one
-  **Exchange table**: Remove FK Currency Id

**ATTENTION**: I was thinking on Currency table, seems like we don't need that table at all, we will have just a enum, feel free to drop that column and remove relation related to the table

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+https://drive.google.com/file/d/1OogCU_qmvH-yhxRAk1pWMAEwSOxFxhkL/view?usp=sharing    
+https://drive.google.com/file/d/1OogCU_qmvH-yhxRAk1pWMAEwSOxFxhkL/view?usp=sharing   with deposit table
```
Comment (ghost) [inline README.md]: looks the same as link above, but with wrong FK UserId on card table

Comment (ghost) [reply README.md]: You should paste an image, not a link. Find out how to do that

Comment (Nazar865) [reply README.md]: resolved

File: README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+![image](https://user-images.githubusercontent.com/91699763/166444077-613cefef-6149-4cb2-b132-3df8c52f1292.png)
```
Comment (ghost) [inline README.md]: Card: where is the FK for user?
Remove underlines for tables that handles many-to-many

Comment (Nazar865) [reply README.md]: resolved

Comment (ghost) [reply README.md]: Looks good, one think to note: mark Ids as a FK where it is needed

Comment (Nazar865) [reply README.md]: resolved

**Pull request merged**